### PR TITLE
marvelmind_ros2_release: 1.0.2-1 in 'humble/distribution.yaml' [bloom]

### PR DESCRIPTION
**Pull request by smoker771**
Increasing version of package(s) in repository `marvelmind_ros2_release` to `1.0.2-1`:

- upstream repository: https://github.com/MarvelmindRobotics/marvelmind_ros2_upstream.git
- release repository: https://github.com/MarvelmindRobotics/marvelmind_ros2_release_repo.git
- distro file: `humble/distribution.yaml`
- bloom version: `0.11.1`
- previous version for package: `null`

## marvelmind_ros2

```
* project files added
* first commit
* Contributors: Marvelmind Robotics, smoker771
```

### CODE REVIEW

**Comment by tfoote** [comment]
[Holding for Humble sync](https://discourse.ros.org/t/preparing-for-humble-sync-on-july-15th/26452)

**Comment by jacobperron** [comment]
This is no longer on hold.

@smoker771 could you resolve the Git conflicts?

File: humble/distribution.yaml [hunk 1]
```diff
@@ -1978,6 +1978,14 @@ repositories:
...
       version: dashing-devel
     status: developed
+  marvelmind_ros2_release:
+    release:
+      packages:
+      - marvelmind_ros2
+      tags:
+        release: release/humble/{package}/{version}
+      url: https://github.com/MarvelmindRobotics/marvelmind_ros2_release_repo.git
+      version: 1.0.2-1
```
[the remark this answers is not in the document]
Comment (smoker771) [reply humble/distribution.yaml:1988]: Hello. Do I need to execute bloom again and specify the link to the source? Or I can click 'Resolve conflicts'  and edit the file manually?

[the remark this answers is not in the document]
Comment (jacobperron) [reply humble/distribution.yaml:1988]: I would do the latter: resolve conflicts and edit the file manually.

**Pull request closed without merging**